packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -164,13 +164,18 @@ export const stripExcessParens = (context: CompletionContext) => (option: Comple
 	return option;
 };
 
-export const getDefaultArgs = (doc?: DocMetadata): unknown[] => {
-	return doc?.args?.map((arg) => arg.default).filter(Boolean) ?? [];
+export const getDefaultArgs = (doc?: DocMetadata): string[] => {
+	return (
+		doc?.args
+			?.filter((arg) => !arg.optional)
+			.map((arg) => arg.default)
+			.filter((def): def is string => !!def) ?? []
+	);
```

Comment:
Nit - With `reduce` we can do a single pass instead of three.

Reply:
Personally find this more readable and the args are always short so I'll keep this if that's OK